core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
actually, they must be separated, I tried linking them together but maintaining separate gradient settings, it still did not work, so we'll have to keep it separate.